sunpy/database/tests/test_tables.py

```diff
@@ -96,6 +113,108 @@ def test_tag_hashability():
 
 @pytest.mark.flaky(reruns=5)
 @pytest.mark.online
+def test_entries_from_fido_search_result(fido_search_result):
+    entries = list(entries_from_fido_search_result(fido_search_result))
+    # 65 entries for 8 instruments in fido_search_result
+    assert len(entries) == 65
+    # First 2 entries are from lyra
+    assert entries[0] == DatabaseEntry(
+        source='Proba2', provider='esa', physobs='irradiance',
+        fileid='http://proba2.oma.be/lyra/data/bsd/2012/01/01/lyra_20120101-000000_lev2_std.fits',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=np.nan, wavemax=np.nan,
+        instrument='lyra')
+    # 54 entries from EVE
+    assert entries[2] == DatabaseEntry(
+        source='SDO', provider='LASP', physobs='irradiance',
+        fileid='EVE_L1_esp_2012001_00',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        instrument='EVE', size=-1.0,
+        wavemin=0.1, wavemax=30.400000000000002)
+    # 2 entries from goes
+    assert entries[56] == DatabaseEntry(
+        source='nasa', provider='sdac', physobs='irradiance',
+        fileid='http://umbra.nascom.nasa.gov/goes/fits/2012/go1520120101.fits',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=np.nan, wavemax=np.nan,
+        instrument='goes')
+    # 1 entry from noaa-indices
+    assert entries[58] == DatabaseEntry(
+        source='sdic', provider='swpc', physobs='sunspot number',
+        fileid='ftp://ftp.swpc.noaa.gov/pub/weekly/RecentIndices.txt',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=np.nan, wavemax=np.nan,
+        instrument='noaa-indices')
+    # 1 entry from noaa-predict
+    assert entries[59] == DatabaseEntry(
+        source='ises', provider='swpc', physobs='sunspot number',
+        fileid='http://services.swpc.noaa.gov/text/predicted-sunspot-radio-flux.txt',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=np.nan, wavemax=np.nan,
+        instrument='noaa-predict')
+    # 2 entries from norh
+    assert entries[60] == DatabaseEntry(
+        source='NAOJ', provider='NRO', physobs="",
+        fileid=("ftp://anonymous:data@sunpy.org@solar-pub.nao.ac.jp/"
+                "pub/nsro/norh/data/tcx/2012/01/tca120101"),
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=17634850.470588233, wavemax=17634850.470588233,
```

Comment:
I think these numbers have gone through a round of GHz -> nm conversion, which is probably the source of the precision.